src/codemap.rs

```diff
@@ -0,0 +1,39 @@
+use syntax::codemap::{BytePos, CodeMap, Span};
```

Comment:
Missing copyright notice.

Also could you add a comment for what belongs in this file please? I think there is potential for confusion with FileMap and so forth.